lasagne/layers/merge.py

```diff
@@ -303,8 +303,9 @@ class ElemwiseMergeLayer(MergeLayer):
     Parameters
     ----------
     incomings : a list of :class:`Layer` instances or tuples
-        the layers feeding into this layer, or expected input shapes,
-        with all incoming shapes being equal
+        the layers feeding into this layer, or expected input shapes.
+        All input shapes must be equal or broadcastable. A dimension is
+        broadcastable if all non None dimensions are the same size or 1.
```

Comment:
Hmm, I don't think this makes it really clear, and it's also wrong since we've got auto-cropping. Maybe write:
```
Unless `cropping` is given, all shapes must be equal,
except for dimensions that are undefined (``None``)
or broadcastable (``1``).
```

 "with all incoming shapes being equal or broadcastable to the same size".